content/en/architectures/adobe/index.md

```diff
@@ -0,0 +1,300 @@
+---
+title: Scaling Adobe’s Service Delivery Foundation with a Cell-based Architecture
+date: 2024-10-11
+org_name: Adobe
+org_team: Developer Platforms
+org_url: https://www.adobe.com/
+contact: Vikram Sethi
+email: vik.set@gmail.com
+org_description: |
+
```

Comment:
Could you add a brief description of Adobe? 
If you have something from like an internal branding guide that would work well :+1: